statler_api/models.py

```diff
@@ -52,6 +52,10 @@ class PlayListDAO(models.Model):
     title = models.CharField(max_length=256)
     url_title = models.CharField(max_length=32)
 
+    # See https://docs.djangoproject.com/en/dev/topics/db/models/#intermediary-manytomany
+    # for documentation on this many-to-many pattern.
+    plays = models.ManyToManyField(PlayDAO, through='PlayListEntryDAO')
```

Comment:
This tells django that the PlayListEntryDAO is being used for a many-to-many relationship. Which allows a playListDAO's plays to be accessed with .plays instead of going through the PlayListEntryDAO